scripts/js/build_js.mjs

```diff
@@ -4,7 +4,7 @@ import fs from 'fs';
 import path from 'path';
 
 const OUTPUT = 'npm_dist/';
-const HEADER = `// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+const HEADER = `// Copyright (C) 2021-2022 Parity Technologies (UK) Ltd.
```

Comment:
We could replace the right year always with the current year?

Reply:
Yeap, will adjust so we never need to touch again.